src/resources/views/shipping-method/_form.blade.php

```diff
@@ -67,6 +68,39 @@
 
 <hr>
 
+<div class="mb-3 row">
+    <label class="col-form-label col-form-label-sm col-md-2">{{ __('Shipping Category') }}</label>
+    <div class="col-md-10">
+        {{ Form::select('shipping_category_id', $shippingCategories->pluck('name', 'id'), null, [
+                'class' => 'form-select form-select-sm' . ($errors->has('shipping_category_id') ? ' is-invalid': ''),
+                'placeholder' => __('--'),
+                'x-model' => 'selectedShippingCategory'
+           ])
+        }}
+        @if ($errors->has('shipping_category_id'))
+            <div class="invalid-feedback">{{ $errors->first('shipping_category_id') }}</div>
+        @endif
+    </div>
+</div>
+
+<div class="mb-3 row" x-show="isCategorySelected">
+    <label class="col-form-label col-form-label-sm col-md-2">{{ __('Match Products') }}</label>
+    <div class="col-md-10">
+        @foreach($shippingCategoryMatchingConditions as $key => $value)
+            <div class="form-check form-check-inline {{ $errors->has('shipping_category_matching_condition') ? 'is-invalid' : '' }}">
+                {{ Form::radio('shipping_category_matching_condition', $key, $shippingMethod->shipping_category_matching_condition && $shippingMethod->shipping_category_matching_condition->value() == $key, ['id' => "shipping_category_matching_condition_$key", 'x-model' => 'shippingCategoryMatchingCondition', 'class' => 'form-check-input' . ($errors->has('shipping_category_matching_condition') ? ' is-invalid': '')]) }}
+                <label class="form-check-label" for="shipping_category_matching_condition_{{ $key }}">{{ $value }}</label>
+            </div>
+        @endforeach
```

Comment:
Since the `UNUSED` has been added after this PR; The unused can be filtered out, since the null value is handled behind the scenes.

```suggestion
        @foreach($shippingCategoryMatchingConditions as $key => $value)
            @unless(is_null($key))
            <div class="form-check form-check-inline {{ $errors->has('shipping_category_matching_condition') ? 'is-invalid' : '' }}">
                {{ Form::radio('shipping_category_matching_condition', $key, $shippingMethod->shipping_category_matching_condition && $shippingMethod->shipping_category_matching_condition->value() == $key, ['id' => "shipping_category_matching_condition_$key", 'x-model' => 'shippingCategoryMatchingCondition', 'class' => 'form-check-input' . ($errors->has('shipping_category_matching_condition') ? ' is-invalid': '')]) }}
                <label class="form-check-label" for="shipping_category_matching_condition_{{ $key }}">{{ $value }}</label>
            </div>
            @endunless
        @endforeach
```